pkg/asset/installconfig/kubevirt/validation.go

```diff
@@ -0,0 +1,120 @@
+package kubevirt
+
+import (
+	"errors"
+	"fmt"
+	"net"
+
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/kubevirt"
+	"github.com/openshift/installer/pkg/types/kubevirt/validation"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+)
+
+// Validate executes kubevirt specific validation
+func Validate(ic *types.InstallConfig, clientBuilder ClientBuilderFuncType) error {
+	kubevirtPlatformPath := field.NewPath("platform", "kubevirt")
+
+	if ic.Platform.Kubevirt == nil {
+		return errors.New(field.Required(
+			kubevirtPlatformPath,
+			"validation requires a KubeVirt platform configuration").Error())
```

Comment:
This is a coding error and should not be presented to the user as a field error.

Reply:
Removed this validation, as it should never be true - also validated by the caller